megamek/src/megamek/common/Mech.java

```diff
@@ -7082,6 +7159,100 @@ public boolean addDualCockpit() {
         setCockpitType(COCKPIT_DUAL);
         return true;
     }
+	
+    public boolean addSuperheavyIndustrialCockpit() {
+        if (getEmptyCriticals(LOC_HEAD) < 5) {
+            return false;
+        }
+        addCritical(LOC_HEAD, 0, new CriticalSlot(CriticalSlot.TYPE_SYSTEM,
+                SYSTEM_LIFE_SUPPORT));
+        addCritical(LOC_HEAD, 1, new CriticalSlot(CriticalSlot.TYPE_SYSTEM,
+                SYSTEM_SENSORS));
+        addCritical(LOC_HEAD, 2, new CriticalSlot(CriticalSlot.TYPE_SYSTEM,
+                SYSTEM_COCKPIT));
+        addCritical(LOC_HEAD, 4, new CriticalSlot(CriticalSlot.TYPE_SYSTEM,
+                SYSTEM_SENSORS));
+        addCritical(LOC_HEAD, 5, new CriticalSlot(CriticalSlot.TYPE_SYSTEM,
+                SYSTEM_LIFE_SUPPORT));
+        setCockpitType(COCKPIT_SUPERHEAVY_INDUSTRIAL);
+        return true;
+    }
+	
+    public boolean addSuperheavyCommandConsole() {
```

Comment:
does this need the same check as above in `addSuperheavyIndustrialCockpit` regarding the number of empty head criticals?

Reply:
Its core is same as addCommandConsole(), and it doesn't check for the slots either. It seems that it was intended to not check if there is a remaining slot if the console requires all the slots on head.